skimage/filters/thresholding.py

```diff
@@ -235,17 +235,22 @@ def threshold_local(image, block_size, method='gaussian', offset=0,
 
     return thresh_image - offset
 
-
-def threshold_otsu(image, nbins=256):
+def threshold_otsu(image=None, nbins=256, *, histogram=None):
```

Comment:
Please find another name here as it conflicts with the `histogram` function imported from `exposure` at the top of the file (another solution would be to use it as `exposure.histogram` in the whole file, but it's probably better to choose another name, maybe just `hist`?)

Reply:
Good catch! It was called `hist` earlier. So we can stick to that shorter name.